src/test/kotlin/no/nav/syfo/CreateArenaDialogNotatTest.kt

```diff
@@ -77,6 +82,22 @@ internal class CreateArenaDialogNotatTest {
         arenaDialogNotat.svarReferanse shouldBeEqualTo "OD1812186729156"
     }
 
+    @Test
+    internal fun `Default to 0 when tss id is empty string`() {
+        val healthcareProfessional =  XMLHealthcareProfessional()
+        val avsender = createAvsenderLege(FASTLEGE_FNR,"", healthcareProfessional)
```

Comment:
Pirk: mellomrom mellom `FASTLEGE_FNR` og `""`? Tilsvarende for linje 96?

Reply:
Blæh! Det er intellij som legger inn en noen ganger hjelpsom tekst som gjør det vanskelig å se.. I fix!